server/src/main/java/org/spine3/server/storage/memory/InMemoryCommandStorage.java

```diff
@@ -41,24 +41,22 @@
     @Override
     public void write(CommandId id, CommandStorageRecord record) {
         checkNotNull(id);
```

Comment:
We also need `Identifiers.checkValid()` for checking `EventId`, `CommandId`, `AggregateId`, `ProcessManagerId`.